src/Serilog.Settings.Configuration/Settings/Configuration/Assemblies/AssemblyFinder.cs

```diff
@@ -16,12 +16,19 @@ protected static bool IsCaseInsensitiveMatch(string text, string textToFind)
 
         public static AssemblyFinder Auto()
         {
-            // Need to check `Assembly.GetEntryAssembly()` first because 
-            // `DependencyContext.Default` throws an exception when `Assembly.GetEntryAssembly()` returns null
-            if (Assembly.GetEntryAssembly() != null && DependencyContext.Default != null)
+            try
             {
-                return new DependencyContextAssemblyFinder(DependencyContext.Default);
+                // Need to check `Assembly.GetEntryAssembly()` first because 
+                // `DependencyContext.Default` throws an exception when `Assembly.GetEntryAssembly()` returns null
+                if (Assembly.GetEntryAssembly() != null && DependencyContext.Default != null)
+                {
+                    return new DependencyContextAssemblyFinder(DependencyContext.Default);
+                }
             }
+            catch (NotSupportedException) when (typeof(object).Assembly.Location is "") // bundled mode detection
+            {
+            }
+            
```

Comment:
should this actually throw instead of ignoring silently ? or writing a message to SelfLog or something with a hint about what needs to be done ?

Reply:
It throws currently :) AFAIK there is no good workaround for single file deployments, except using `Using` section. I've updated the docs.

The problem with writing to `SelfLog` is it will show message with correct configuration (because of using `Using` section or not using external assemblies).

Reply:
oh right, nevermind then !